src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -4396,6 +4396,18 @@ subroutine Summary_carbonflux(this, &
             this%hrv_gresp_storage_to_litter_patch(p)      + &
             this%hrv_gresp_xfer_to_litter_patch(p)
 
+       if (use_mimics_decomp) then
+          ! Calculate ligninNratio for leaves and fine roots
```

Comment:
> Do the ligninNratioAvg calculation at the p level and then call p2c for ligninNratioAvg.

@wwieder I have now realized (or possibly remembered) that cwd variables are tracked at the column level only. So I'm calculating ligninNratio(leaf) and (root) at the p level and the rest at the column level a few lines down.

Reply:
this makes sense, thanks for clarifying @slevisconsulting